### DISCUSSION TITLE
`showLineNumbers` by default

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [facebook/docusaurus#8748] [Q&A] [answered]
Hey there,

Is there any way, by default in all code blocks, to show the number of lines (`showLineNumbers`)? (Sometimes I forget to add 😅)

Thanks any help :)

**Comment by slorber** [comment, marked as answer]
We don't have this option currently but I would be happy to add it.

Already discussed here: https://github.com/facebook/docusaurus/pull/7007#discussion_r844545282

**Reply by 201flaviosilva** [reply]
Ohhh, nice, thanks :)

**Reply by pondorasti** [reply]
That would be great! 🚀
